src/main/java/com/telefonica/iot/cygnus/sinks/OrionMongoBaseSink.java

```diff
@@ -122,9 +122,6 @@ public void configure(Context context) {
         LOGGER.debug("[" + this.getName() + "] Reading configuration (collection_prefix=" + collectionPrefix + ")");
         shouldHash = context.getBoolean("should_hash", false);
         LOGGER.debug("[" + this.getName() + "] Reading configuration (should_hash=" + shouldHash + ")");
-        this.rowAttrPersistence = context.getString("attr_persistence", "row").equals("row");
```

Comment:
Now, `rowAttrPersistence` must be declared at `OrionMongoSink`, not at `OrionMongoBaseSink`